test/intree_test.js

```diff
@@ -145,6 +145,16 @@ suite('intree config', () => {
       tasks: [],
     });
 
+  buildConfigTest(
```

Comment:
Really there are a couple cases we should test maybe?

1. The case you have here
2. An exclude overridden by an include

Reply:
I have added it in the docs that include takes precedence over exclude

Reply:
jhford march 23 through april 2

Reply:
Agreed, but once we state something in the docs it is nice to assert it in tests as well, right?

Reply:
Yes. I'll make the test right away :)